cmd/upgrade/operatorroles/cmd.go

```diff
@@ -442,16 +442,16 @@ func upgradeMissingOperatorRole(missingRoles map[string]*cmv1.STSOperator, clust
 
 func getRoleName(cluster *cmv1.Cluster, missingOperator *cmv1.STSOperator) string {
 	operatorIAMRoles := cluster.AWS().STS().OperatorIAMRoles()
-	rolePrefix := ""
+	rolePrefix := ocm.ComputeDefaultOperatorRolesPrefix(cluster.Name())
```

Comment:
We cannot do this

Reply:
@pvasant: Could you please detail a bit more why we cannot do this?
We are only defining the default prefix here.
The nominal case is still to extract the prefix from the operator role name in the code that follows. If the prefix cannot be extracted, the code is currently ending in exception while with my change the code will continue to run normally and create the new operator roles shipped in the upgrade with the default prefix.